easybuild/easyconfigs/f/FragGeneScan/FragGeneScan-1.31-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'FragGeneScan'
+version = '1.31'
+
+homepage = 'https://omics.informatics.indiana.edu/FragGeneScan/'
+description = "FragGeneScan is an application for finding (fragmented) genes in short reads."
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s%(version)s.tar.gz']
+checksums = ['cd3212d0f148218eb3b17d24fcd1fc897fb9fee9b2c902682edde29f895f426c']
+
+builddependencies = [('binutils', '2.31.1')]
+
+buildopts = 'CC="$CC" CFLAG="$CFLAGS" fgs && chmod -R go+rx *.pl train example'
+
+files_to_copy = ['FragGeneScan', 'run_FragGeneScan.pl', 'example', 'train']
```

Comment:
Doesn't `run_FragGeneScan.pl` indicate that this ought to have a `Perl` dependency?

Reply:
Good catch. It had hard-coded shebang and was using system Perl. I've added shebang fix and Perl deps.
I didn't pay attention to that, as I was updating from older configfile.